src/components/ChatTranscript/LineItem.js

```diff
@@ -28,7 +28,11 @@ const LineItem = (props: Props) => {
   ) : null
 
   const contentMarkup = body ? (
-    <span dangerouslySetInnerHTML={{ __html: newlineToHTML(body) }} />
+    <span
+      dangerouslySetInnerHTML={{
+        __html: newlineToHTML(escapeHTML(body)),
```

Comment:
@tompedals I have a thought on tidying this up a bit! We have a neat util function in `react-utils` called [compose](https://github.com/helpscout/react-utils/blob/master/docs/compose.md). 

We can create a function like:

```js
const enhanceBody = compose(newlineToHTML, escapeHTML)

...

dangerouslySetInnerHTML={{
  __html: enhanceBody(body)
}}
```

or


```js
const htmlBody = compose(newlineToHTML, escapeHTML)(body)

...

dangerouslySetInnerHTML={{
  __html: htmlBody
}}
```

This pattern can be repeated for the other uses as well.

💖